.github/workflows/release.yml

```diff
@@ -0,0 +1,22 @@
+name: Release
+on:
+  push:
+    branches:
+      - master
+      - next
+      - beta
+      - "*.x" # maintenance releases branches
+
+jobs:
+  release:
+    name: release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+      - run: npm ci
```

Comment:
what would you think of using https://github.com/bahmutov/npm-install for the install step? automatically implements [caching in an optimized way](https://glebbahmutov.com/blog/do-not-let-npm-cache-snowball/) for us

Reply:
sure, once we add lock files?

Reply:
lockfiles shouldnt have much of an impact specifically for this. internally, it runs `npm ci`, just like you did, but also configures caching. the caching should work regardless of lockfile